src/Domain/CoordinateMaps/FocallyLiftedSide.hpp

```diff
@@ -0,0 +1,312 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <limits>
+#include <optional>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeArray.hpp"
+#include "Utilities/TypeTraits/RemoveReferenceWrapper.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace domain::CoordinateMaps::FocallyLiftedInnerMaps {
+/*!
+ * \brief A FocallyLiftedInnerMap that maps a 3D unit right cylindrical shell
+ *  to a volume that connects portions of two spherical surfaces.
+ *
+ * \details The domain of the map is a 3D unit right cylinder with
+ * coordinates \f$(\bar{x},\bar{y},\bar{z})\f$ such that
+ * \f$-1\leq\bar{z}\leq 1\f$ and \f$1\leq \bar{x}^2+\bar{y}^2 \leq
+ * 4\f$.  The range of the map has coordinates \f$(x,y,z)\f$.
+ *
+ * Consider a sphere with center \f$C^i\f$ and radius \f$R\f$ that is
+ * intersected by two planes normal to the \f$z\f$ axis located at
+ * \f$z = z_\mathrm{L}\f$ and \f$z = z_\mathrm{U}\f$, with
+ * \f$z_\mathrm{L} < z_\mathrm{U}\f$.
+ * `Side` provides the following functions:
+ *
+ * ### operator()
```

Comment:
Thanks for noticing the `operator()` (it used to be `operator()` in an older version but I forgot to change the dox)

Not sure what you are asking about individual function documentation.  Are you suggesting to replace the big doxygen block with smaller doxygen blocks, one for each individual function? (And then still allow these blocks to refer to each other?)

Reply:
Yeah, I was suggesting that the functions be smaller blocks attached directly to the functions they describe. Though, I see the point about cross-references. Would something like 'equation (N) from the documentation for `other_function()`' be a reasonable solution?

Reply:
Not sure if that would be clearer, because then people would be jumping back and forth between functions.   And another more practical matter: none of the other CoordinateMaps have the dox split in this way.  So should it be done for all of them?

Reply:
Ah, I see -- I didn't realize this was chosen to be uniform with the other CoordinateMaps. In that case, I'm fine with it as-is for now, and we can reform them all at once if we conclude the function-by-function documentation would be an improvement in the future.

Reply:
Ok.  It would be nice if there were an easy way for doxygen to link each function to a section of a larger documentation block.  Don't know if that is possible or easy.